build-system/tasks/check-sourcemaps.js

```diff
@@ -165,12 +167,16 @@ function checkSourcemapMappings(sourcemapJson) {
     log(helpMessage);
     throwError('Found mapping for incorrect code');
   }
-  if (generatedCodeColumn != expectedFirstLineColumn || sourceCodeColumn != 0) {
```

Comment:
Why are we deleting the column check?

Reply:
We're already testing `firstLineFile === expectedFirstLineFile`, which is the unmapping part we care about. This `generatedCodeColumn` doesn't actually matter (we only care that it's the first mapping segment and points to the correct spot). Between ESM and regular builds, the actual generated column is different (`"use strict"` isn't necessary in ESM).